packages/valory/skills/abstract_round_abci/tests/test_utils.py

```diff
@@ -158,21 +149,14 @@ def test_to_int_positive() -> None:
     assert to_int(542, 2) == 54200
 
 
-@pytest.mark.skip
-def test_fuzz_to_int() -> None:
+@given(
+    st.floats(width=64, allow_nan=False, allow_infinity=False),
+    st.integers(min_value=0, max_value=20),
+)
+@pytest.mark.skip(reason="https://github.com/valory-xyz/open-autonomy/issues/1614")
+def test_fuzz_to_int(estimate: float, decimals: int) -> None:
```

Comment:
raised an issue because this will not pass the test, however neither is it used anywhere in our code base #1614

Reply:
Needs to be moved to https://github.com/valory-xyz/apy-oracle and renamed to close the issue. Let's address this later since it is a low priority.